tdesign/src/Layout/ContentHeader/ContentHeader.tsx

```diff
@@ -9,27 +10,60 @@ export interface IContentHeaderProps {
 }
 
 const ContentHeader = ({ children, extraStyle = {} }: IContentHeaderProps) => {
+  const theme = useTheme();
+
+  const contentHeaderBannerStyle: SxProps<Theme> = {
+    display: "flex",
+    flexWrap: "wrap",
+    marginBottom: "1rem",
+    justifyContent: "space-between",
+    alignContent: "flex-start",
+    alignItems: "flex-end",
+    h2: {
+      color: "flambeeDarkGray.light",
+      letterSpacing: "0.005em",
+      fontWeight: "600",
+      display: "inline",
+      margin: 0,
+    },
+    [theme.breakpoints.down("sm")]: {
+      flexDirection: "column",
+      alignItems: "flex-start !important",
+    },
+    ".repo-list-title": {
+      color: "flambeeDarkGray.dark",
+      fontWeight: "600",
+      fontSize: "20px",
+      lineHeight: "20px",
+      letterSpacing: "0.005em",
+      display: "inline",
+      margin: 0,
+      marginBottom: "1rem",
+      [theme.breakpoints.up("sm")]: { flex: "1 0 50%" },
+    },
+    ".repo-list-link": {
+      color: "flambeeRed.main",
+      [theme.breakpoints.down("sm")]: { order: 99 },
+      [theme.breakpoints.up("sm")]: {
+        flex: "1 0 50%",
+        textAlign: "end",
+      },
+    },
+    ".repo-list-subtitle": {
+      color: "flambeeDarkGray.light",
+      fontSize: "14px",
+      lineHeight: "24px",
+      fontWeight: "normal",
+      marginTop: "0.5rem",
+      [theme.breakpoints.up("sm")]: { flex: "0 1 100%" },
+    },
+  };
+
   const headerContainerStyle: SxProps<Theme> = {
     marginBottom: "2rem",
-    // color: (theme: Theme) =>
-    //   theme.palette.getContrastText(
-    //     theme.palette.surfaces.light.background.main
-    //   ),
-    // ...extraStyle,
+    ...extraStyle,
     ".content-header--banner": {
-      // padding: "0.5em",
-      marginBottom: { md: "2rem" },
-      display: "flex",
-      justifyContent: "space-between",
-      alignContent: "center",
-      alignItems: "flex-end",
-      ...(extraStyle.hasOwnProperty(".content-header--banner")
```

Comment:
I'm worried this feature (allowing consumers to override partially override `content-header--banner`) of the previous styles is not supported in new styles

Reply:
Let me know if this is intentional :slightly_smiling_face:

Reply:
Looks good now :+1: Thanks for bringing it back